src/renderer/components/views/projects/SkeletonProjectsView.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useMemo } from 'react';
+import { Card, SkeletonBar } from '@/components';
+
+const renderSkeletonBars = (count: number) => {
+  return Array.from({ length: count }, (_, index) => <SkeletonBar key={index} />);
+};
+
+const SkeletonProjectsView: React.FC = () => {
```

Comment:
this can be called SkeletonView or similar and moved to components/layout. it cna be used for both the projectsTab and unitsTab